specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/examples/RandomNumbers-example.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "vaultBaseUrl": "https://myhsm.managedhsm.azure.net/",
+    "api-version": "7.3-preview",
+    "parameters": {
+      "properties": {
```

Comment:
"properties" should not be in the example - this is causing two of the validation errors. It should just be:
```json
"parameters": {
  "bytesLength": 4
}
```